<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeMachine</class>
 <widget class="QMainWindow" name="TimeMachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/newPrefix/icons/backup.png</normaloff>:/newPrefix/icons/backup.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="auto_checkbox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>165</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Back Up Automatically</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_folders">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>395</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Folders</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_disk">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>150</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushBUutton{
border-radius:25px;
}</string>
    </property>
    <property name="text">
     <string>Select Backup Disk...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_txt">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>210</y>
      <width>340</width>
      <height>108</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time Machine Keeps:&lt;/p&gt;&lt;p&gt;* Local snapshots as space permits&lt;/p&gt;&lt;p&gt;The oldest backups are deleted when disk becomes full.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_usb_name">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>50</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_last_backup">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>80</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Last backup: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_next_backup">
    <property name="geometry">
     <rect>
      <x>434</x>
      <y>100</y>
      <width>251</width>
      <height>30</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Next backup: &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_schedule">
    <property name="geometry">
     <rect>
      <x>505</x>
      <y>395</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Schedule</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_last_backup_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/backup.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_last_backup_3">
    <property name="geometry">
     <rect>
      <x>306</x>
      <y>50</y>
      <width>81</width>
      <height>91</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/newPrefix/restore_48.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons/timemachine.qrc"/>
 </resources>
 <connections/>
</ui>
